src/components/RelicScorerTab.jsx

```diff
@@ -53,6 +53,20 @@ export default function RelicScorerTab() {
   const [scorerForm] = Form.useForm()
   window.scorerForm = scorerForm
 
+  useEffect(() => {
+    const url = window.location.search
+    if (url) {
+      const params = new URLSearchParams(url)
+      const id = params.get('uid')
+      onFinish({ scorerId: id, source: 'page loaded with id in url' })
```

Comment:
Do we need this source field here? Looks unused

Reply:
not needed no, I just added it so that I can see in console that it did actually load by reading the url